File: php/src/Application/Command/ReleaseMission/ReleaseMission.php
<?php
declare(strict_types=1);

namespace App\Application\Command\ReleaseMission;

final class ReleaseMission
{
    /** @var string */
    private $plateauId;
    /** @var string */
    private $roverId;

    public function __construct(string $plateauId, string $roverId)
    {
        $this->plateauId = $plateauId;
        $this->roverId   = $roverId;
    }

    public function plateauId(): string
    {
        return $this->plateauId;
    }

    public function roverId(): string
    {
        return $this->roverId;
    }
}

File: csharp/MarsRoverKata/Application/Command/ReleaseMission.cs
using System;

namespace MarsRoverKata.Application.Command
{
    public class ReleaseMission
    {
        private string _plateauId;
        private string _roverId;

        public ReleaseMission(string plateauId, string roverId)
        {
            _plateauId = plateauId;
            _roverId = roverId;
        }

    }
}
